conferences/2018/javascript.json

```diff
@@ -415,12 +415,13 @@
     "country": "U.S.A."
   },
   {
-    "name": "JSDay Italy",
```

Comment:
jsDay happens in Spain and Uruguay too
2017 jsDay Italy https://2017.jsday.it/
2017 jsDay Spain http://2017.jsday.es/
2017 JSDay Uruguay http://day.js.uy/

I'll retain the name jsDay Italy and merge the code

Reply:
jsDay is a registered trade mark. But I agree for the sake of the list :)